typesystem/schema/smoke_test_struct.h

```diff
@@ -85,6 +85,11 @@ CURRENT_STRUCT(FullTest) {
   CURRENT_FIELD_DESCRIPTION(primitives, "A structure with a lot of primitive types.");
   CURRENT_FIELD(v1, std::vector<std::string>);
   CURRENT_FIELD(v2, std::vector<Primitives>);
+#if 0
```

Comment:
This is what I am not adding support for (yet), full schema export yields some `UNKNOWN` types.